### PR TITLE
goodie_cheat_sheets.md - update directory for cheat sheets

### PR DESCRIPTION
**Pull request by rahiel**
I think this is what's meant.

### CODE REVIEW

File: duckduckhack/goodie/goodie_cheat_sheets.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ After cloning the Goodies repository, you simply need to add **one JSON file** t
 
 To do this in Codio, use the left-hand panel to enter the `/zeroclickinfo-goodies` repository directory. Then use the file tree to click into the `/share/goodie/cheat_sheets` directory. Then click the **json** folder. 
 
-Up in the **File menu**, click **"Create New File"**, and enter the name of your cheat sheet as a JSON file (make sure it's saving to the `cheat_sheets` directory). For example, if the trigger for your cheat sheet is "gopro cheat sheet" create a new file called `gopro.json`.
+Up in the **File menu**, click **"Create New File"**, and enter the name of your cheat sheet as a JSON file (make sure it's saving to the `json` directory). For example, if the trigger for your cheat sheet is "gopro cheat sheet" create a new file called `gopro.json`.
```
Comment (moollaza) [inline duckduckhack/goodie/goodie_cheat_sheets.md]: I think it should actually be `cheat_sheets/json` to be unambiguously clear?

/cc @talsraviv

Comment (rahiel) [inline duckduckhack/goodie/goodie_cheat_sheets.md]: I updated it. (My editor also deleted all trailing whitespace.)

**Comment by moollaza** [comment]
@rahiel it looks like there's a merge conflict now as we just merged another PR that modified this page. Can you please pull in the latest updates to your branch so we can merge this PR?

**Comment by rahiel** [comment]
@moollaza done!

**Comment by moollaza** [comment]
@rahiel thanks a lot! this LGTM :+1:

Thanks a lot for contributing (we really appreciate it :smiley:). We invite you to stick around and comment/help on other PR's & Issues or even submit an Instant Answer!

**Pull request merged**